### PR TITLE
Create README.md on organisation level

### PR DESCRIPTION
**Pull request by CheeseTastisch**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+1. 👉 Concept: In this phase, a concept for QuillORMs **migrations only** approach is developed.
+2. Migrations: In this phase the migration engine is developed, 
+to actually convert the created migrations to a database structure.
+3. ORM: In this phase the actual ORM and therefore mapping between the database and the models is developed.
+This is the base for the model generation, which will only rely on the ORM to do the heavy lifting.
+4. Model generation: In this phase the model generation is developed, which will generate the models based on the
+migrations.
+
+## How to contribute?
+If you want to contribute to QuillORM, you can do so by creating a pull request.
```
Comment (SIMULATAN) [inline profile/README.md:23]:
```suggestion
If you want to contribute to QuillORM, you can do so by creating a pull request, opening an issue, starting a discussion or participating in existing ones.
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+However, in the current phase of development, where a concept is developed, we try to keep the team as small as possible,
+to keep the basic view of QuillORM in sight and make development as fast as possible.
+
+## Concept
+You can view the current concept at [Concept.md](https://github.com/CheeseTastisch/QuillORM/blob/main/concept/Concept.md).
```
Comment (SIMULATAN) [inline profile/README.md:29]: wrong link - use relative links instead

**Review by SIMULATAN — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 2 above)
Comment (CheeseTastisch) [reply profile/README.md:29]: Maybe we should leave that link out for now?
I'm currently not sure, wheater we should create the concept in the .github repository or create an own repository for that belonging.

Any ideas on it?

Comment (SIMULATAN) [reply profile/README.md:29]: dunno lmao

Comment (CheeseTastisch) [reply profile/README.md]: Fixed in new commit

**Review by SIMULATAN — APPROVED** [review]
i approve of this
yes

**Pull request merged**